dex-it/src/test/scala/com/wavesplatform/it/sync/networking/NetworkIssuesTestSuite.scala

```diff
@@ -40,6 +45,33 @@ class NetworkIssuesTestSuite extends WsSuiteBase with HasToxiProxy {
     clearOrderBook()
   }
 
+  "DEXClient should place orders despite of short time disconnect from network" in {
+    val orders = (0 to 100).map { i =>
+      mkOrder(alice, wavesUsdPair, OrderType.SELL, 1.waves, 100 + i)
+    }
+
+    def placeOrders: Future[Unit] =
+      Future
+        .inSeries(orders)(dex1.asyncApi.place(_))
+        .transform {
+          case Success(_) => Try()
+          case Failure(e) => Try(e)
+        }
+
+    Await.result(
+      for {
+        _         <- placeOrders.zip { Future(blocking(wavesNode1.reconnectToNetwork(500, 500))) }
+        orderBook <- dex1.asyncApi.orderBook(wavesUsdPair)
+      } yield {
+        orderBook.asks should have size 100
+        orders.foreach(dex1.api.waitForOrderStatus(_, OrderStatus.Accepted))
```

Comment:
Please move this outside the `Await.result`, because it:
1. Synchronous and doesn't depend on values in for comprehension
2. Affects the timeout of 2 minutes

Reply:
fixed